scripts/cleanup_kubeflow_ci.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# This script cleans up ingress related compute engine resources like backend-services,
+# forwarding rules, health-checks, etc. from kubeflow-ci project which
+# do not belong to the kubeflow-testing cluster
+
+set -xe
+
+PROJECT="--project=kubeflow-ci"
```

Comment:
nit: PROJECT should be the project name. The "--project" should be in the command.

Reply:
Done